packages/opc-styles/variables/_1-definitions.scss

```diff
@@ -6,9 +6,28 @@ $prefix: opc !default;
 $USE-FALLBACK: true;
 
 /// Global variables for Gap/ Spacing in layout
-$opc-global--spacer--sm:  0.5rem; // 8px
-$opc-global--spacer--md:  1rem;   // 16px
-$opc-global--spacer--lg:  1.5rem; // 24px
-$opc-global--spacer--xl:  2rem;   // 32px
-$opc-global--spacer--2xl: 3rem;   // 48px
-$opc-global--spacer--3xl: 4rem;   // 64px
+$opc-global--spacer--sm:              0.5rem; // 8px
+$opc-global--spacer--md:              1rem;   // 16px
+$opc-global--spacer--lg:              1.5rem; // 24px
+$opc-global--spacer--xl:              2rem;   // 32px
+$opc-global--spacer--2xl:             3rem;   // 48px
+$opc-global--spacer--3xl:             4rem;   // 64px
+
+/// Global variables for font-weight
+$opc-global--fontWeight--light:       300;
+$opc-global--fontWeight--normal:      500;
+$opc-global--fontWeight--semi-bold:   600;
+$opc-global--fontWeight--bold:        700;
+
+/// Heading font-size
+$opc-global--heading--xxl:            3rem;    /// 48px
+$opc-global--heading--xl:             2.5rem;  /// 40px
+$opc-global--heading--one:            2.25rem; /// 36px
+$opc-global--heading--two:            1.75rem; /// 28px
+$opc-global--heading--three:          1.5rem;  /// 24px
+$opc-global--heading--four:           1.25rem; /// 20px
+
+/// Body Text font-size
+$opc-global--text--lg:                1.12rem; /// 18px
+$opc-global--text--md:                1rem;    /// 16px
+$opc-global--text--sm:                0.8rem;  /// 14px
```

Comment:
@gisumit can we rename these to opc-global--FontSize--lg format instead of naming them as heading and text. and also add 0.75rem  (12px) as opc-globbal--FontSize--xs for small sizes. 
can u convert these global--heading to global--FontSize--<xs/sm,md,lg,xl,2xl,3xl>

Reply:
yes updated the PR